test/integration/job/job_test.go

```diff
@@ -3304,8 +3307,8 @@ func TestElasticIndexedJob(t *testing.T) {
 // also faster to track, as they need less API calls.
 func BenchmarkLargeIndexedJob(b *testing.B) {
 	closeFn, restConfig, clientSet, ns := setup(b, "indexed")
-	restConfig.QPS = 100
-	restConfig.Burst = 100
+	restConfig.QPS = restConfigQPS
```

Comment:
such a high QPS makes sense for integration, as we are trying to measure correctness.

But in a benchmark, we want to make sure that the Job controller is fast enough when the QPS is limited.

Reply:
That makes sense.
I reverted those, and added comments the reason.
PTAL.

Reply:
Makes sense, thanks for the suggestion and the update.